tempesta_fw/http.h

```diff
@@ -107,11 +107,13 @@ enum {
 #define TFW_HTTP_CC_MIN_FRESH		0x00000020
 #define TFW_HTTP_CC_OIFCACHED		0x00000040
 /* Response only CC directives. */
+
 #define TFW_HTTP_CC_MUST_REVAL		0x00000100
 #define TFW_HTTP_CC_PROXY_REVAL		0x00000200
 #define TFW_HTTP_CC_PUBLIC		0x00000400
 #define TFW_HTTP_CC_PRIVATE		0x00000800
 #define TFW_HTTP_CC_S_MAXAGE		0x00001000
+#define TFW_HTTP_CC_STALE		0x00001100 /*is stale, but pass*/
```

Comment:
This is not a `Cache-Control` header field directive, so this is not the right place for the flag. It is a response flag. Perhaps `TFW_HTTP_RESP_STALE` is a better name.